lib/cli.zsh

```diff
@@ -281,8 +281,8 @@ multi == 1 && length(\$0) > 0 {
 
   local zdot="${ZDOTDIR:-$HOME}"
   awk "$awk_script" "$zdot/.zshrc" > "$zdot/.zshrc.new" \
-  && command mv -f "$zdot/.zshrc" "$zdot/.zshrc.bck" \
-  && command mv -f "$zdot/.zshrc.new" "$zdot/.zshrc"
+  && command cp "$(echo $~zdot/.zshrc(:A))" "$zdot/.zshrc.bck" \
+  && command mv -f "$zdot/.zshrc.new" "$(echo $~zdot/.zshrc(:A))"
```

Comment:
Hi Ben, you were actually very close: you don't need to use glob modifiers, since these are also applicable to parameters and they don't use a subcommand:
```suggestion
  && command cp -f "${${:-"$zdot/.zshrc"}:A}" "$zdot/.zshrc.bck" \
  && command mv -f "$zdot/.zshrc.new" "${${:-"$zdot/.zshrc"}:A}"
```
I believe this would be less ugly-looking by introducing a new `local` variable:
```zsh
local zshrc="${${:-"${zdot}/.zshrc"}:a}"
awk "$awk_script" "$zshrc" > "${zdot}/.zshrc.new" \
&& command cp -f "$zshrc" "${zdot}/.zshrc.bck" \
&& command mv -f "${zdot}/.zshrc.new" "$zshrc"
```
The `mv -f` was intentional here, but I left the `cp`. Since the `.bck` file is not getting deleted though, it should be forced to override the previous ones if existing.